heudiconv/convert.py

```diff
@@ -202,14 +205,16 @@ def prep_conversion(sid, dicoms, outdir, heuristic, converter, anon_sid,
         clear_temp_dicoms(item_dicoms)
 
     if bids:
```

Comment:
We should log a warning to users if this file exists before entering the context manager, maybe something like:

> Existing lockfile found in {path to LOCKFILE} - waiting the lock to be released. To set a timeout limit, set the HEUDICONV_FILELOCK_TIMEOUT environmental variable to a value in seconds. If this process hangs, it may require a manual deletion of the {LOCKFILE}.